main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+var (
+	tempFileName = "trivy-go-plugin-temp.json"
+)
+
+func main() {
+	trivyCommand := os.Args[1 : len(os.Args)-1]
+	tempFilePath := filepath.Join(os.TempDir(), tempFileName)
```

Comment:
now we can add the custom flag, eg. `--plugin-output` for the result.
it must be a required option.

to Trivy we shouldn't pass this flag